src/LoRa.h

```diff
@@ -73,6 +73,7 @@ class LoRaClass : public Stream {
   void setSyncWord(int sw);
   void enableCrc();
   void disableCrc();
+  uint8_t crcOnPayload();
```

Comment:
I'm thinking this could be renamed to something more consistent with: https://github.com/sandeepmistry/arduino-LoRa/blob/master/src/LoRa.h#L43-L46

Maybe: `packetHasCrc()`?

Also, for the return type type is `bool` a better option?

Thoughts?

Reply:
isCrcPresent?

Normally a bool variable may start with is...?

Reply:
As far as I know, in Arduino IDE `bool` is same than `uint8_t`. I have habit to use uint8_t, but I have nothing against bool. bool is just little bit more abstract.

I think too that crcOnPayload is little problematic function name. It is accurate, but it doesn't indicate what it means to reader. Both your suggestions are better than "crcOnPayload".

Reply:
Hi @mcgurk, if you can address these minor changes I'll merge this PR and include in the next release I'd like to get out this week. Thanks for this contribution and keen eyed reading of the datasheet.

Reply:
Hmm.. This is my first pull request ever :D. What I have to do? Edit my branch? I try.

Reply:
Yeah you just push additional commits to your branch and they will show up here.